docs/src/components/version-control.tsx

```diff
@@ -11,9 +11,11 @@ import { useLocation } from "@docusaurus/router";
 import { cn } from "../lib/utils";
 import { BetaIcon, StableIcon, TriggerIcon, VersionLabel } from "./icons/icon";
 
+import FeatureVersioning from "../../feature-versioning.json";
+
 const versions = [
-  { value: "stable", label: "Stable" },
-  { value: "beta", label: "Beta" },
+  { value: "stable", label: "Stable (v0)" },
+  { value: "beta", label: "Beta (v1)" },
```

Comment:
Took the liberty to add this. I think it helps the user to know which one they are looking at.